controllers/istiorevision/istiorevision_controller.go

```diff
@@ -409,6 +417,18 @@ func (r *Reconciler) isRevisionReferencedByWorkloads(ctx context.Context, rev *v
 	log := logf.FromContext(ctx)
 	nsList := corev1.NamespaceList{}
 	nsMap := map[string]corev1.Namespace{}
+	// if we're referenced by an in-use revisionTag, we're done. we can outsource the checking to the IstioRevisionTagController
```

Comment:
Is the revision tag controller also checking pods? What if you have created a tag but are still using the rev label and not the tag name to label your namespace?

Reply:
The only thing the tag adds, if it exists, is an additional condition for the namespace. Now it could be either `istio.io/rev: <rev>` or `istio.io/rev: <tag>`.

Reply:
actually... I was previously checking the pods but that doesn't make sense, as their annotation will always have the actual revision name, not the tag. so we don't need to check pods in the revisiontag controller

Reply:
okay I removed the annotation checking, but I'm now checking the label to identify injection intent